docs_user/modules/proc_adopting-the-compute-service.adoc

```diff
@@ -13,26 +13,71 @@ To adopt the {compute_service_first_ref}, you patch an existing `OpenStackContro
 * You have completed the previous adoption steps.
 * You have defined the following shell variables. Replace the following example values with the values that are correct for your environment:
 ----
-$ alias openstack="oc exec -t openstackclient -- openstack"
+alias openstack="oc exec -t openstackclient -- openstack"
+
+DEFAULT_CELL_NAME="cell3"
+RENAMED_CELLS="cell1 cell2 $DEFAULT_CELL_NAME"
+----
++
+The `default` cell takes a new name from `DEFAULT_CELL_NAME`.
+In a multi-cell adoption scenario, it may retain its original 'default' name as well.
+
+A standalone TripleO only creates a default cell, so you should define that instead:
++
+----
+DEFAULT_CELL_NAME="cell1"
+RENAMED_CELLS="cell1"
 ----
++
 
 .Procedure
 
 . Patch the `OpenStackControlPlane` CR to deploy the {compute_service}:
 +
 [NOTE]
 This procedure assumes that {compute_service} metadata is deployed on the top level and not on each cell level. If the {OpenStackShort} deployment has a per-cell metadata deployment, adjust the following patch as needed. You cannot run the metadata service in `cell0`.
+To enable local cells metadata services, set `spec.nova.template.cellTemplates.cell*.metadataServiceTemplate.enable` in `OpenStackControlPlane` CR.
 +
 [source,yaml]
 ----
-$ oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+rm -f celltemplates
+for CELL in $(echo $RENAMED_CELLS); do
+  cat >> celltemplates << EOF
+        ${CELL}:
+          hasAPIAccess: true
+          cellDatabaseAccount: nova-$CELL
+          cellDatabaseInstance: openstack-$CELL
+          cellMessageBusInstance: rabbitmq-$CELL
+          metadataServiceTemplate:
+            enabled: false # enable here to run it in a cell instead
+            override:
+                service:
+                  metadata:
+                    annotations:
+                      metallb.universe.tf/address-pool: internalapi
+                      metallb.universe.tf/allow-shared-ip: internalapi
+                      metallb.universe.tf/loadBalancerIPs: 172.17.0.$(( 79 + ${CELL##*cell} ))
+                  spec:
+                    type: LoadBalancer
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=true
+          conductorServiceTemplate:
+            customServiceConfig: |
+              [workarounds]
+              disable_compute_service_check_for_ffu=true
+EOF
+done
+
+cat > oscp-patch.yaml << EOF
 spec:
   nova:
     enabled: true
     apiOverride:
       route: {}
     template:
       secret: osp-secret
+      apiDatabaseAccount: nova-api
```

Comment:
nit: feels like an unrelated change

Reply:
I used https://github.com/openstack-k8s-operators/nova-operator/blob/42b7b8417b480886f9d80f0d279d12abb9ccf58e/config/samples/nova_v1beta1_nova-multi-cell.yaml#L9 as the source of truth